syntax_and_semantics/operators.md

```diff
@@ -21,71 +38,201 @@ struct Vector2
+| Range           | `..`, `...`                                          |
+| Conditional     | `?:`                                                 |
+| Assignment      | `=`, `[]=`, `+=`, `&+=`, `-=`, `&-=`, `*=`, `&*=`, `/=`, `//=`, `%=`, `\|=`, `&=`, `^=`, `**=`, `<<=`, `>>=`, `\|\|=`, `&&=` |
+
+## Operators explained
+### Arithmetic operators
+#### Unary
+
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `+`  | positive | `+1` | yes |
+| `&+` | positive (overflowing) | `&+1` | yes |
+| `-`  | negative | `-1` | yes |
+| `&-` | negative (overflowing) | `&-1` | yes |
+
+#### Multiplicative
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `**` | exponentiation | `1 ** 2` | yes |
+| `&**` | exponentiation (overflowing) | `1 &** 2` | yes |
+| `*` | multiplication | `1 * 2` | yes |
+| `&*` | multiplication (overflowing) | `1 &* 2` | yes |
+| `/` | division | `1 / 2` | yes |
+| `//` | floor division | `1 // 2` | yes |
+| `%` | modulus | `1 % 2` | yes |
+
+#### Additive
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `+` | addition | `1 + 2` | yes |
+| `&+` | addition (overflowing) | `1 &+ 2` | yes |
+| `-` | subtraction | `1 - 2` | yes |
+| `&-` | subtraction (overflowing) | `1 &- 2` | yes |
+
+### Other unary operators
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `!` | inversion | `!true` | no |
+| `~` | binary complement | `~1` | yes |
+
+### Shifts
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `<<` | shift left, append | `1 << 2`, `STDOUT << "foo"` | yes |
+| `>>` | shift right | `1 >> 2` | yes |
+
+### Binary
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `&` | binary AND | `1 & 2` | yes |
+| `\|` | binary OR | `1 \| 2` | yes |
+| `^` | binary XOR | `1 ^ 2` | yes |
+
+### Equality
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `==` | equals | `1 == 2` | yes |
+| `!=` | not equals | `1 != 2` | yes |
+| `=~` | pattern match | `"foo" =~ /fo/` | yes |
+| `!~` | no pattern match | `"foo" !~ /fo/` | yes |
+| `===` | [case equality](case.html) | `` | yes |
+
+### Comparison
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `<` | less | `1 < 2` | yes |
+| `<=` | less or equal | `1 <= 2` | yes |
+| `>` | greater | `1 > 2` | yes |
+| `>=` | greater or equal | `1 >= 2` | yes |
+| `<=>` | comparison | `1 <=> 2` | yes |
+
+### Logical
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `&&` | logical AND | `true && false` | yes |
+| `\|\|` | logical OR | `true || false` | yes |
+
+### Range
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `..` | range | `1..10` | no |
+| `...` | exclusive range | `1...10` | no |
+
+### Splats
+| Operator | Description | Example | Overloadable |
+|---|---|---|---|
+| `*` | splat | `*foo` | no |
+| `**` | double splat | `**foo` | no |
+
+### Block
```

Comment:
I'm not sure the Blocks section belongs here. Because they are not operators but syntax that use symbols.

Reply:
Theoretically they're are operators and so are `.`, `::` and others. I'm not yet sure how to handle all this.